.eslintrc.json

```diff
@@ -24,5 +24,33 @@
             }
         ],
         "eol-last": 2
-    }
+    },
+    "import/order": [
+        "error",
+        {
+          "groups": ["builtin", "external", "internal"],
+          "pathGroups": [
+            {
+              "pattern": "react",
+              "group": "external",
+              "position": "before"
+            }
+          ],
+          "pathGroupsExcludedImportTypes": ["react"],
+          "newlines-between": "always",
+          "alphabetize": {
+            "order": "asc",
+            "caseInsensitive": true
+          }
+        }
+      ],
+      "import/prefer-default-export": "off",
+      "no-nested-ternary": "off",
+      "@typescript-eslint/no-unused-vars": [
+        "error",
+        {
+          "argsIgnorePattern": "^_"
+        }
+      ]
+    
```

Comment:
Added these config because it would work without additional extension installed on the vscode or whatever.